### PR TITLE
Adds event.target to shouldServe function

### PR DESCRIPTION
**Pull request by tamb**
#11

### CODE REVIEW

File: src/pill.js [hunk 1]
```diff
@@ -158,7 +158,7 @@ export default function pill(selector, options) {
   }
 
   function onClick (e) {
-    if (e.target.nodeName !== 'A') {
+    if (e.target.nodeName !== 'A' || e.target.dataset.pillIgnore) {
```
Comment (rumkin) [inline src/pill.js]: Replace data-attribute check with callback arguments extension.

**Review by rumkin — CHANGES_REQUESTED** [review]
Use `e.target` as the second argument to shouldServe callback below.

File: src/pill.js (hunk 1 above)
Comment (tamb) [reply src/pill.js]: updated

File: readme.md [hunk 2]
```diff
@@ -121,43 +121,43 @@ All page-related content should be located inside `#content`. It could be styles
 ## API
 
 ### `pill()`
-```
+```js
```
Comment (rumkin) [inline readme.md]: Remove all `js` code block languages. It's my own pseudocode for type description and not a JS at all.

**Review by rumkin — CHANGES_REQUESTED** [review]

File: readme.md (hunk 2 above)
Comment (tamb) [reply readme.md]: Got it.

**Review by rumkin — APPROVED** [review]

**Pull request merged**